lib/cli/package.json

```diff
@@ -20,7 +20,7 @@
     "dist": "rm -rf ../../dist/libs/cli && npm run build && cp -R ./bin ../../dist/libs/cli && cp -R ./resources ../../dist/libs/cli && cp -R ./templates ../../dist/libs/cli && cp ./package.json ../../dist/libs/cli"
   },
   "dependencies": {
-    "@alfresco/js-api": ">=8.0.0-alpha.6-0",
```

Comment:
For cli please keep the older version here as it tried to fetch this version during the build, it won't work with latest version

Reply:
done